Fixed #33724 -- Doc'd exclude argument changes in model validation. (#15726)
django/django: docs/ref/models/instances.txt

    The ``validate_constraints`` argument was added.

.. versionchanged:: 4.1

>     An ``exclude`` value is now converted to the ``set`` instead of ``list``.

Reviewer: ```suggestion
    An ``exclude`` value is now converted to a ``set`` rather than a ``list``.
```

Author: Updated :+1:
